processors/config/kalSimpTuple_cfg.py

```diff
@@ -121,48 +139,57 @@
 vtx.parameters["kinkRelCollLcio"] = ''
 vtx.parameters["trkRelCollLcio"] = 'KFTrackDataRelations'
 
-cvtx.parameters["debug"] = 0
-cvtx.parameters["vtxCollLcio"]     = 'TargetConstrainedV0Vertices_KF'
-cvtx.parameters["vtxCollRoot"]     = 'TargetConstrainedV0Vertices_KF'
-cvtx.parameters["partCollRoot"]    = 'ParticlesOnCVertices_KF'
-cvtx.parameters["kinkRelCollLcio"] = ''
-cvtx.parameters["trkRelCollLcio"]  = 'KFTrackDataRelations'
-
-
 vtxgbl.parameters["debug"] = 0
 vtxgbl.parameters["vtxCollLcio"]     = 'UnconstrainedV0Vertices'
 vtxgbl.parameters["vtxCollRoot"]     = 'UnconstrainedV0Vertices'
-vtxgbl.parameters["partCollRoot"]    = 'ParticlesOnUVertices'
+vtxgbl.parameters["partCollRoot"]    = 'ParticlesOnVertices'
 vtxgbl.parameters["kinkRelCollLcio"] = 'GBLKinkDataRelations'
 vtxgbl.parameters["trkRelCollLcio"]  = 'TrackDataRelations'
 
 cvtxgbl.parameters["debug"] = 0
 cvtxgbl.parameters["vtxCollLcio"]     = 'TargetConstrainedV0Vertices'
 cvtxgbl.parameters["vtxCollRoot"]     = 'TargetConstrainedV0Vertices'
-cvtxgbl.parameters["partCollRoot"]    = 'ParticlesOnCVertices'
+cvtxgbl.parameters["partCollRoot"]    = 'ParticlesOnVertices'
```

Comment:
There should be a "C" on this one, we could also using something more than just "C", but the important thing is it has a different name than the unconstrained vtx.